examples/playground/index.html

```diff
@@ -5,6 +5,7 @@
   <title>deck.gl Playground</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href='https://unpkg.com/maplibre-gl@3.6.0/dist/maplibre-gl.css' rel='stylesheet' />
+  <link href="https://unpkg.com/deck.gl@^9.0.0/dist/stylesheet.css" rel='stylesheet' />
```

Comment:
- Nice if widgets could dynamically load this stylesheet, maybe as an option
- Or maybe add affordance to the JSON syntax to allow for stylesheets to be specified.